Octokit/Clients/Enterprise/EnterpriseAdminStatsClient.cs

```diff
@@ -0,0 +1,115 @@
+﻿using System;
+using System.Diagnostics.CodeAnalysis;
+using System.Threading.Tasks;
+
+namespace Octokit
+{
+    /// <summary>
+    /// A client for GitHub's Enterprise Admin Stats API
+    /// </summary>
+    /// <remarks>
+    /// See the <a href="https://developer.github.com/v3/enterprise/admin_stats/">Enterprise Admin Stats API documentation</a> for more information.
+    ///</remarks>
+    public class EnterpriseAdminStatsClient : ApiClient, IEnterpriseAdminStatsClient
+    {
+        public EnterpriseAdminStatsClient(IApiConnection apiConnection)
+            : base(apiConnection)
+        { }
+
+        /// <summary>
+        /// Gets GitHub Enterprise statistics (must be Site Admin user).
+        /// </summary>
+        /// <remarks>
+        ///https://developer.github.com/v3/enterprise/admin_stats/#get-statistics
+        /// </remarks>
+        /// <returns>The <see cref="AdminStats"/> collection for the requested <see cref="AdminStatsType"/> type.</returns>
+        [SuppressMessage("Microsoft.Globalization", "CA1308:NormalizeStringsToUppercase")]
+        public async Task<AdminStats> GetStatistics(AdminStatsType type)
```

Comment:
At first I wrote all the code assuming the admin stats endpoint would always send back the "collection of stats" regardless of what you asked for, and if you only asked for say "repos" then the other items wouldnt be populated (and thus would end up null in my c# object).

But when doing the integration testing what i actually found was that only the "All" method returns back the outside collection (ie collection of stats entries), and all of the other "specific" requests only send back their specific json fragment and not the outer collection.

Since I'd already coded it to have an Enum for the type of stats being requested, I took the "easy" way out and just populated the specific node in the AdminStats collection that is returned by the ALL stats call.  This meant there is 1 function, 1 type of return class, and an Enum to specify the type of stats you want.

The alternative was to remove the enum and add specific functions for every type of request.  

As an end user I felt happy enough with the 1 function/1 return class/enum to specify what I want (most people would probably just request ALL stats anyway im assuming).  I felt like that was preferable to adding 15 methods 
eg
```
Task<AdminStats> GetAllStatistics()
Task<AdminStatsRepo> GetRepoStatistics()
Task<AdminStatsPullRequest> GetPullRequestStatistics()
```
and so on.

If you think it's more fitting with the rest of octokit.net, let me know and i can change it.